source/reference/hyperlinks.txt

```diff
@@ -0,0 +1,81 @@
+.. _style-guide-hyperlinks:
+
+==========
+Hyperlinks
+==========
+
+You can link out to other sites, internal pages or sections using inline
+hyperlinks and cross-referenced hyperlinks.
+
+Inline Hyperlinks
+-----------------
+
+Use inline hyperlinks when you want to link to a full URL.
+
+Example
+~~~~~~~
+
+.. code-block:: rst
+
+   Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Rendered View
+`````````````
+
+Here is the `MongoDB Go Driver Documentation <https://www.mongodb.com/docs/drivers/go/current/>`__.
+
+Cross-Referenced Hyperlinks
+---------------------------
+
+Use cross-referenced hyperlinks when you want to link to another page in
+docs. This page can be in the same repo, or another within the same
+organization.
+
+Procedure
+~~~~~~~~~
+
+.. procedure::
+   :style:  normal
+
+   .. step:: If the hyperlink doesn't already exist, in the line before where you want to link to, create the cross-referenced hyperlink.
+
+      If the hyperlink is in a different repo, add the rendered docs URL
+      landing page and append ``objects.inv``. Add the URL to the
+      ``intersphinx`` array in the ``snooty.toml`` file.
+
+      For example, if you want to use hyperlinks in the manual, you'd
+      add this to the ``intersphinx`` array:
+
+      .. code-block:: rst
+
+         https://www.mongodb.com/docs/manual/objects.inv
+
+   .. step:: Reference the cross-referenced hyperlink.  
+
+Example
+```````
```

Comment:
Is this Example supposed to fall under the previous step? If so, there's probably a way to make that clearer. If not, ignore this comment!

Reply:
I added some introductory text here to avoid confusion among other folks too :)